src/librustc_incremental/calculate_svh.rs

```diff
@@ -69,7 +70,7 @@ impl<'a, 'tcx> SvhCalculate for TyCtxt<'a, 'tcx, 'tcx> {
         // to avoid hashing the AttrId
         for attr in &krate.attrs {
             debug!("krate attr {:?}", attr);
-            attr.node.value.hash(&mut state);
+            attr.meta().hash(&mut state);
```

Comment:
Do you have name suggestions? `value()`? `meta_item()` (which is dangerously close to `meta_item_list()`?